econml/dml.py

```diff
@@ -42,68 +71,130 @@ class _RLearner(LinearCateEstimator):
         take an extra second argument (the treatment residuals).  Predict, on the other hand,
         should just take the features and return the constant marginal effect.
 
-    n_splits: int
-        The number of splits to use when fitting the first-stage models.
+    n_splits: int, or a custom cross validator object
+        The number of splits to use when fitting the first-stage models.  If this is an integer,
+        then splits will be generated by `StratifiedKFold(shuffle=True)` with that number of splits
+        if treatment is discrete and `KFold(shuffle=True)` otherwise.  Otherwise, the argument will
+        be treated as a custom cross validator.
```

Comment:
We should update here on how the custom cross validator is called, i.e. that n_splits.split(X, T) is called to generate the CrossFit splits. should be useful for the user without the need to look at code.

Reply:
Fixed in next push.